endpoints/publicProxy/http.go

```diff
@@ -106,6 +106,9 @@ func newServiceProxy(cfg *Config, ctx ziti.Context) (*httputil.ReverseProxy, err
 	director := proxy.Director
 	proxy.Director = func(req *http.Request) {
 		director(req)
+		if cfg.NoForwarder {
+			req.Header["X-Forwarded-For"] = nil
```

Comment:
It's a pretty well-known header, but it appears twice in zrok. It'd be nice to use a const imo